Algorithm.Framework/Risk/TrailingStopRiskManagementModel.cs

```diff
@@ -27,15 +27,15 @@ namespace QuantConnect.Algorithm.Framework.Risk
     public class TrailingStopRiskManagementModel : RiskManagementModel
     {
         private readonly decimal _maximumDrawdownPercent;
-        private Dictionary<Symbol, decimal> _trailingHighs = new Dictionary<Symbol, decimal>();
+        private readonly Dictionary<Symbol, decimal> _trailing = new Dictionary<Symbol, decimal>();
 
         /// <summary>
         /// Initializes a new instance of the <see cref="TrailingStopRiskManagementModel"/> class
         /// </summary>
         /// <param name="maximumDrawdownPercent">The maximum percentage relative drawdown allowed for algorithm portfolio compared with the highest unrealized profit, defaults to 5% drawdown per security</param>
         public TrailingStopRiskManagementModel(decimal maximumDrawdownPercent = 0.05m)
         {
-            _maximumDrawdownPercent = -Math.Abs(maximumDrawdownPercent);
+            _maximumDrawdownPercent = Math.Abs(maximumDrawdownPercent);
```

Comment:
Have you considered a different `maximumDrawdownPercent` for long and short positions (e.g.: `maximumDrawdownPercentLong` and `maximumDrawdownPercentLong`)? It would give more flexibility to this model. The parameters could be `decimal?` and if we use
```csharp
var model = TrailingStopRiskManagementModel(0.05m, null)
```
would turn off the trailing stop for short positions and 
```csharp
var model = TrailingStopRiskManagementModel(null, 0.05m)
```
would turn off the trailing stop for long positions.

Reply:
I think that in most cases, the same logic is used to generate both long and short signals, i.e mirrored reversed, and this usually makes these insights equally relevant. I believe there are many other cases where this separation could be useful. To keep the things simple, I 've only done testing with uniform trailing stop for both long/short. Also possible to note, that sometimes it may work better to trail the stops by various indicators that change over time. I am more supporter of this concept.